File: client-mvp/app/src/main/java/ua/meugen/android/rxjavatests_client/view/MainView.java
package ua.meugen.android.rxjavatests_client.view;

import com.hannesdorfmann.mosby3.mvp.MvpView;

import java.util.List;

/**
 * @author meugen
 */

public interface MainView extends MvpView {

    void showData(List<String> data);

    void showProgress();
}

File: client-kotlin/app/src/main/java/ua/meugen/android/rxjavatests_client/view/MainView.kt
package ua.meugen.android.rxjavatests_client.view

import com.hannesdorfmann.mosby3.mvp.MvpView

/**
 * @author meugen
 */

interface MainView : MvpView {

    fun showData(data: List<String>)

    fun showProgress()
}
